src/constants/base.js

```diff
@@ -1,10 +1,9 @@
 import { searchResultListColumns } from '@folio/stripes-authority-components';
 
-export const MAIN_PANE_HEIGHT = '609px';
+export const MAIN_PANE_HEIGHT = '575px';
```

Comment:
I think the previous value is correct. The ident is present now.
![image](https://user-images.githubusercontent.com/77053927/197507277-95b1ba5b-1b19-4ca8-90a6-89a73e3cebad.png)


![image](https://user-images.githubusercontent.com/77053927/197506930-5ccc575b-fc67-43a3-904b-b447d63bff57.png)